Source/WebCore/inspector/agents/InspectorDOMAgent.cpp

```diff
@@ -3095,4 +3095,19 @@ Protocol::ErrorStringOr<void> InspectorDOMAgent::setAllowEditingUserAgentShadowT
     return { };
 }
 
+Protocol::ErrorStringOr<Ref<JSON::Object>> InspectorDOMAgent::getMediaStats(Protocol::DOM::NodeId nodeId)
+{
+    Protocol::ErrorString errorString;
+
+    Element* element = assertElement(errorString, nodeId);
+    if (!element)
+        return makeUnexpected(errorString);
+
+    if (!element->isMediaElement())
+        return makeUnexpected(errorString);
```

Comment:
Won't this return an empty string?

Reply:
Ah, it will. Maybe that's fine?

Reply:
Okay, Added an explicit error string in this case.